### DISCUSSION TITLE
Deye Inverter

### DISCUSSION THREAD

**Discussion by straga** [Sleeper85/esphome-jk-bms-can#3] [General]
For example:
Deye shows : Bat voltage:  52.25 V
BMS shows: 52.47 V

Somebody have Deye, what voltage shows yours inverter?

**Comment by Sleeper85** [comment]
As you can see in the photo of the dashboard that I posted. The shunt voltage is exactly the same as that of the BMS and the Deye too but with only one decimal.

I have calibrated the BMS voltage to match with the shunt, tested with a good voltmeter.

Calibration of the BMS must be done while the battery is at rest, never during a charge or big discharge.

Concerning the charging voltage requested from the Deye, I noticed 0.1V difference between what is asked of it and the charging voltage it actually uses. This is why I use 55.3V in the app so that Deye actually charges with 55.2V. This is strange.

That said, at rest, the voltage displayed on Deye must be the same as the BMS.

**Comment by straga** [comment]
![telegram-cloud-photo-size-4-5785262034101845355-y](https://github.com/Sleeper85/esphome-jk-bms-can/assets/34532/e6931d15-a6cf-4139-9c4e-5a4450f8d241)

![telegram-cloud-photo-size-4-5785262034101845356-y](https://github.com/Sleeper85/esphome-jk-bms-can/assets/34532/b2de4275-5b7b-4dbf-8f14-23dffb2668b1)

Multimetr shows 52.49, BMS 52.43 use Fluke Calibration Multimetr.
Deye shows 52.22

I want order Vectron Smart Shunt for all time monitoring.

Also for JK BMS have reset SOC:
https://github.com/Louisvdw/dbus-serialbattery/blob/master/etc/dbus-serialbattery/config.default.ini

**Comment by straga** [comment]
Real Volt - Multimeter Fluke: 53.43, 
JKBMS 53.43, 
Shunt : 53.43. 
Float voltate 53.6+0.1 = 53.7
Deye shows battery voltage 53.20
That normal ? Or need some tune ?

**Reply by b1ggi** [reply]
https://diysolarforum.com/threads/deye-weird-voltage-behavior-when-time-of-use-is-enabled.80908/
im also in search for a solution to this problem...
